# Dynamic Window Approach :Velocity Limit
min_linear : 0.0
max_linear : 1.2
min_angular_deg : -90.0
max_angular_deg : 90.0
# Dynamic Window Approach : Step Value
predict_step : 15
sampling_time : 0.1     # Predicted time[s] = predict_step[step] * sampling_time[s]
velocity_step : 7.0
angle_velocity_step : 30.0
# Dynamic Window Approach : Weight
weight_heading : 3.0
weight_obstacle : 1.0
weight_velocity : 1.5
# Dynamic Window Approach : Cost Distance
obstacle_cost_radius : 0.25
# Dynamic Window Approach : Display Flag
display_optimal_path : true
display_all_path : true